cluster-autoscaler/cloudprovider/digitalocean/README.md

```diff
@@ -0,0 +1,58 @@
+# Cluster Autoscaler for DigitalOcean
+
+The cluster autoscaler for DigitalOcean scales worker nodes within any
+specified DigitalOcean Kubernetes cluster's node pool. This is part of the DOKS
+offering which can be enabled/disable dynamically for an existing cluster.
+
+# Configuration
+
+The `cluster-autoscaler` dynamically runs based on tags associated with node
+pools. These are the current valid tags:
+
+```
+k8s-cluster-autoscaler-enabled:true
+k8s-cluster-autoscaler-min:3
+k8s-cluster-autoscaler-max:10
+```
+
+The syntax is in form of `key:value`.
+
+* If `k8s-cluster-autoscaler-enabled:true` is absent or
+  `k8s-cluster-autoscaler-enabled` is **not** set to `true`, the
+  `cluster-autoscaler` will not process the node pool by default.
+* To set the minimum number of nodes to use `k8s-cluster-autoscaler-min`
+* To set the maximum number of nodes to use `k8s-cluster-autoscaler-max`
+
+
+If you don't set the minimum and maximum tags, node pools will have the
+following default limits:
+
+```
+minimum number of nodes: 1
+maximum number of nodes: 200
```

Comment:
That is significantly different UX from other providers - I think all (?) existing providers require explicitly setting at least max limit.

Reply:
This is the default limit for a given node group right now. It's out upper limit. I can change it to 50 or something else if that makes more sense to you? In the UI (cloud.digitalocean.com) we're going to expose a UI that will have sensible pre-default values.

Reply:
I don't have any specific recommendation for the value. I was referring to the fact that there is a default at all, rather than an explicit requirement to specify a value.

Reply:
I see what you mean. This is just a way of limiting the hard maximum we allow. The user experience will be different and our API will validate requests.